docs/en/siem/rules-ui-create.asciidoc

```diff
@@ -0,0 +1,141 @@
+* `High`: Signals that require an immediate investigation.
+* `Critical`: Signals that indicate a security incident has occurred.
+.. _Risk score_: A numerical value between 0 and 100 that correlates with the _Severity_ level. General guidelines are:
+* `21` represents low severity.
+* `47` represents medium severity.
+* `73` represents high severity.
+* `99` represents critical severity.
+.. For additional options, click *Advanced settings* and fill in any of 
+these fields: 
+... _Timeline template_ (optional): Select a timeline template used when you 
+send a signal created by the rule to the timeline.
++
+TIP: Before you create rules, create and save relevant
+<<timelines-ui,timelines>> so they can be selected here. 
+
+... _Reference URLs_ (optional): References to information that is relevant to 
+the rule. For example, links to relevant background information.
+... _False positives_ (optional): List of common scenarios that may produce 
+false-positive signals.
+... _MITRE ATT&CK^TM^_ (optional): Relevant MITRE framework tactics and techniques.
+... _Tags_ (optional): Words and phrases used to categorize, filter, and search 
+the rule.
+. Click *Continue*.
++
+The *Schedule rule* pane is displayed.
+[role="screenshot"]
+image::schedule-rule.png[]
+. Select how often the rule runs.
+. Optionally, add `look-back` time to the rule. When defined, the rule searches 
+indices with the additional time.
++
+For example, if you set a rule to run every 5 minutes with an additional
+look-back time of 1 minute, the rule runs every 5 minutes but analyses the 
+documents added to indices during the last 6 minutes. This overlap helps ensure 
+signals are issued even when incoming data is delayed.
+. Save the rule with or without activation.
++
+NOTE: When you activate a rule, it is queued and its schedule is determined by 
+its initial run time. For example, if you activate a rule that runs every 5 
+minutes at 14:02, but it does not run until 14:04, it will run again at 14:09.
+
+[[manage-rules-ui]]
+==== Modify existing rules
+
+You can clone, edit, activate, deactivate, and delete rules:
+
+. Go to *SIEM* -> *Detections* -> *Manage signal detection rules*.
+. Do one of the following:
+* Click the actions icon (three dots) and then select the required action.
+* In the *Rule* column, select all the rules you want to act on, and then the 
+required action from the `Batch actions` menu.
+. To activate or deactivate a rule, click the Activate toggle button.
+
+
+[[import-export-rules-ui]]
+==== Import and export rules
+
+. Go to *SIEM* -> *Detections* -> *Manage signal detection rules*.
+. To import rules:
+.. Click *Import rule*.
+.. Drag-and-drop files containing the signal detection rules.
++
+NOTE: Imported rules must be in a `ndjson` file.
+
+. To export rules:
+.. In the *All rules* table, select the rules you want to export.
+.. Select *Batch actions* -> *Export selected*.
++
+NOTE: You cannot export prebuilt rules.
```

Comment:
Not sure if there is going to be separate documentation for the `Rule details` page, but just wanted to note that the export action is also available under the action overflow icon in the top right of the page:

![image](https://user-images.githubusercontent.com/2946766/73142232-b6b00280-4049-11ea-9fb7-d26112103b4b.png)